openedx/core/lib/xblock_builtin/xblock_discussion/tests.py

```diff
@@ -0,0 +1,136 @@
+""" Tests for DiscussionXBLock"""
+import ddt
+import mock
+from nose.plugins.attrib import attr
+from unittest import TestCase
+
+from safe_lxml import etree
+
+from openedx.core.lib.xblock_builtin.xblock_discussion.xblock_discussion import DiscussionXBlock
+from xblock.field_data import DictFieldData
+from xblock.fields import ScopeIds, UNIQUE_ID, NO_CACHE_VALUE
+from xblock.runtime import Runtime
+
+
+@attr('shard2')
+@ddt.ddt
+class DiscussionXBlockImportExportTests(TestCase):
+    """
+    Import and export tests
+    """
+    DISCUSSION_XBLOCK_LOCATION = "openedx.core.lib.xblock_builtin.xblock_discussion.xblock_discussion.DiscussionXBlock"
```

Comment:
Nit: I think you can also just `from ... import DiscussionXBlock` and then use decorators like `@patch.object(DiscussionXBlock, 'load_definition_xml', ...)` which I think is cleaner.